rules_python/piptool.py

```diff
@@ -68,7 +70,12 @@ def extract_packages(package_names):
 
 
 def pip_main(argv):
-    argv = ["--disable-pip-version-check"] + argv
+    # Extract the certificates from the PAR following the example of get-pip.py
+    # https://github.com/pypa/get-pip/blob/430ba37776ae2ad89/template.py#L164-L168
+    cert_path = os.path.join(tempfile.mkdtemp(), "cacert.pem")
+    with open(cert_path, "wb") as cert:
+      cert.write(pkgutil.get_data("pip._vendor.requests", "cacert.pem"))
+    argv = ["--disable-pip-version-check", "--cert", cert_path] + argv
```

Comment:
Yes, this shouldn't be necessary, but ok to add them back in while I investigate more.